docs/_articles/en/lwc/write-lwc.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Write Lightning Web Components in VS Code
+lang: en
+---
+
+The Lightning Web Components Extension for VS Code leverages the editor's built-in code navigation and language features so that you can efficiently build Lightning web components. 
+
+Take advantage of the following Lightning Web Components Extension for VS Code features:
```

Comment:
I feel like the list should be this (which corresponds to the list they have in the general vscode documentation - https://code.visualstudio.com/docs/languages/overview):
Syntax highlighting and bracket matching
Smart code completions (IntelliSense)
Linting and corrections
Code navigation (Go to Definition, Find All References)
View Documentation on Hover